Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py

```diff
@@ -118,6 +118,65 @@ def execute_command(command, args=None):
         return []
 
 
+def test_get_incidents_by_keys():
+    from FindSimilarIncidentsV2 import get_incidents_by_keys
+    res = get_incidents_by_keys({u'name': u'\U0001f489'}, 'created', '2020-10-07T19:49:37.392378+03:00', '7137', 72,
+                                False, '1000', 'status:Closed', 'AND')
+    assert res == [
+        {'CustomFields': {},
+         'account': '',
+         'activated': '0001-01-01T00:00:00Z',
+         'attachment': None,
+         'autime': 1550670443962164000,
+         'canvases': None,
+         'category': '',
+         'closeNotes': '',
+         'closeReason': '',
+         'closed': '0001-01-01T00:00:00Z',
+         'closingUserId': '',
+         'created': '2019-02-20T15:47:23.962164+02:00',
+         'details': '',
+         'droppedCount': 0,
+         'dueDate': '2019-03-02T15:47:23.962164+02:00',
+         'hasRole': False,
+         'id': '1',
+         'investigationId': '1',
+         'isPlayground': False,
+         'labels': [{'type': 'Instance', 'value': 'test'},
+                    {'type': 'Brand', 'value': 'Manual'}],
+         'lastOpen': '0001-01-01T00:00:00Z',
+         'linkedCount': 0,
+         'linkedIncidents': None,
+         'modified': '2019-02-20T15:47:27.158969+02:00',
+         'name': '1',
+         'notifyTime': '2019-02-20T15:47:27.156966+02:00',
+         'occurred': '2019-02-20T15:47:23.962163+02:00',
+         'openDuration': 0,
+         'owner': 'analyst',
+         'parent': '',
+         'phase': '',
+         'playbookId': 'playbook0',
+         'previousRoles': None,
+         'rawCategory': '',
+         'rawCloseReason': '',
```

Comment:
This should definitely be extracted into a variable with a readable name, this test is very hard to read and understand at the moment

Reply:
Done